src/OrchardCore.Modules/OrchardCore.ContentFields/Views/TextField-Monaco.Edit.cshtml

```diff
@@ -49,6 +49,13 @@
             window.addEventListener("submit", function () {
                 textArea.value = editor.getValue();
             });
+            //for preview
+             if(document.getElementById("previewButton")){
```

Comment:
Don't think we need this check, we don't use it anywhere else.

Can you remove for consistency?

Thanks

Reply:
done😊